domain/src/main/java/jinsu/antilog/domain/ExpressionBuffer.kt

```diff
@@ -0,0 +1,53 @@
+package jinsu.antilog.domain
+
+import java.util.LinkedList
+
+class ExpressionBuffer {
+    private val expressionLetters = LinkedList<ExpressionLetter>()
+
+
+    fun addOperand(operand: Operand): String {
+        when (getExpressionLastOrNull()) {
+            is Operand -> {(expressionLetters.last as Operand).addLastLetter(operand)}
```

Comment:
```suggestion
        when (val lastInput = getExpressionLastOrNull()) {
            is Operand -> lastInput.addLastLetter(operand)
```
스마트 캐스팅을 활용해서 이렇게 수정해볼 수 있겠네요 🙂